thirdparty/embree/patches/godot-changes-noexcept-the-sequel.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/thirdparty/embree/common/sys/alloc.h b/thirdparty/embree/common/sys/alloc.h
```

Comment:
This should be merged in `godot-changes-noexcept.patch`, since it's part of the changes needed to disable exceptions.

Reply:
Yeah, I know; it was a last-minute addition and I never bothered merging them back together - will do so now.